src/sentry/sentry_metrics/client/snuba.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Dict, Optional, Sequence, Union
+
+from django.core.cache import cache
+
+from sentry import quotas
+from sentry.sentry_metrics.client.base import GenericMetricsBackend
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+from sentry.testutils.cases import BaseMetricsTestCase
```

Comment:
Do we import from testutils in application code elsewhere? It has a bit of a code smell to me